packages/components/ng-ovh-utils/src/wizard/wizardStep/wizardStep.html

```diff
@@ -33,15 +33,15 @@
 
         <button
             type="button"
-            class="btn"
+            class="btn btn-lg"
```

Comment:
**thought**: I wonder if we could have a cli that migrates bootstrap elements to ui-kit ones ;)

Reply:
I agree. Could be really nice to have that kind of tool.